backend/src/schema/types/type/Category.gql

```diff
@@ -1,13 +1,41 @@
+enum _CategoryOrdering {
+  id_asc
+  id_desc
+  name_asc
+  name_desc
+  slug_asc
+  slug_desc
+  icon_asc
+  icon_desc
+  createdAt_asc
+  createdAt_desc
+  updatedAt_asc
+  updatedAt_desc
+  postCount_asc
+  postCount_desc
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-18T15:15:29Z" title="Monday, November 18th 2019, 4:15:29 pm +01:00">Nov 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,13 +1,41 @@
+enum _CategoryOrdering {
+  id_asc
+  id_desc
+  name_asc
+  name_desc
+  slug_asc
+  slug_desc
+  icon_asc
+  icon_desc
+  createdAt_asc
+  createdAt_desc
+  updatedAt_asc
+  updatedAt_desc
+  postCount_asc
+  postCount_desc
```


any performance issue with this?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-18T20:21:13Z" title="Monday, November 18th 2019, 9:21:13 pm +01:00">Nov 18, 2019</time>_

---


Yes, there are. But we need it for the admin page for tags and categories. Removing this would break the cypress tests.